src/logging.h

```diff
@@ -173,17 +179,31 @@ static inline void LogPrintf(const char* fmt, const Args&... args)
             /* Original format string will have newline so don't add one here */
             log_msg = "Error \"" + std::string(fmterr.what()) + "\" while formatting log message: " + fmt;
         }
-        LogInstance().LogPrintStr(log_msg);
+        const std::string source_location = source_file + ":" + ToString(source_line);
```

Comment:
We could save quite some memory and avoid having to build a `string` here by indexing with a `(const char*, int)` tuple. `__FUNC__` names end up in the `.text` segment after all so a pointer will do.

Reply:
Fixed!